src/main/java/com/demo/testingsonar/TestingSonarApplication.java

```diff
@@ -7,6 +7,7 @@
 public class TestingSonarApplication {
 
     public static void main(String[] args) {
+        String s = "Hola";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "s" local variable. ([squid:S1481](https://sonarcloud.io/organizations/davidalonsobadia-github/rules#rule_key=squid%3AS1481))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.demo%3Atesting-sonar&pullRequest=3&issues=AWPLfAT04fWm9CZEYgFJ&open=AWPLfAT04fWm9CZEYgFJ)

[ISSUE_KEY]: AWPLfAT04fWm9CZEYgFJ